src/components/hamburger/Hamburger.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import style from './Hamburger.module.scss'
+
+export function Hamburger() {
+
+    return (
+        <div className={style.hamburger}>
+            <span className={style.hamburger__line}></span>
+            <span className={style.hamburger__line}></span>
+            <span className={style.hamburger__line}></span>
```

Comment:
don't forget that you can iterate to produce the same items
```suggestion
            {[1,2,3].map((n) => <span className={style.hamburger__line}></span>)}
```